Configuration.props

```diff
@@ -18,10 +18,22 @@
       Project="$(_OutputPath)MonoInfo.props"
       Condition="Exists('$(_OutputPath)MonoInfo.props')"
   />
```

Comment:
Should we use a `Directory.Build.props` instead?

Reply:
In the long run it would be nice to not modify the output paths at all in JI and let XA place things where it wants them.

For example, `Xamarin.Android.Build.Tasks` references `Xamarin.Android.Tools.Bytecode`, so MSBuild will automatically copy `XAT.Bytecode.dll` to the same output path as `XA.Build.Tasks.dll`.

Reply:
That is what `$(UtilityOutputFullPath)` is for, to allow xamarin-android to override the output directory.  This is already used.

Reply:
Using `Directory.Build.props` is a good idea.

Reply:
`Directory.Build.props` is now used.